kie-server-parent/kie-server-tests/kie-server-integ-tests-optaplanner/src/test/java/org/kie/server/integrationtests/optaplanner/OptaplannerIntegrationTest.java

```diff
@@ -140,7 +140,7 @@ public void testGetSolverState() throws Exception {
         assertEquals( SOLVER_1_ID, returnedInstance.getSolverId() );
         assertEquals( SolverInstance.getSolverInstanceKey( CONTAINER_1_ID, SOLVER_1_ID ), returnedInstance.getSolverInstanceKey());
         assertEquals( SolverInstance.SolverStatus.NOT_SOLVING, returnedInstance.getStatus());
-        assertNull( returnedInstance.getScore() );
+        assertNull( returnedInstance.getScoreWrapper() );
```

Comment:
@ge0ffrey This assertion checks whether the score wrapper is null. This happens in cases where the score is null, as there's a little point in returning an empty score wrapper instance.